GitCommands/Patch/PatchManager.cs

```diff
@@ -226,10 +226,11 @@ internal class PatchLine
 
         public PatchLine Clone()
         {
-            var c = new PatchLine();
-            c.Text = Text;
-            c.Selected = Selected;
-            return c;
+            return new PatchLine
```

Comment:
❓ Why would we not want to convert this to a constructor instead?

Reply:
I've added constructor `PatchLine(string text, bool selected = false)`. We cannot make the properties get-only, though `Text` can have a private setter.